### ISSUE TITLE
Quill's Ongoing Accessibility Issues

### ISSUE THREAD

**Issue by qfung** [quilljs/quill#3360]
I was really hoping to use Quill in my project, as it is easier to implement than all the other text editors in the market, however, upon reviewing its accessibility through Google Chrome's WAVE extension, it had produced quite a lot of alarming signs that it is not friendly to users who needed accessibility.
41 empty buttons, coming from the editor itself.
16 Colour contrast errors - that could easily be fixed by the end user through CSS, however, the empty buttons is problematic.

![image](https://user-images.githubusercontent.com/37078772/117017187-a35dff00-acb0-11eb-933e-2213dabd0ad5.png)

**Comment by clairefields15** [comment]
I have all the changes necessary to implement this much needed accessibility work done locally and tested, but I cannot push to the repo to make a PR, can someone help me out?
ERROR: Permission to quilljs/quill.git denied to clairefields15. fatal: Could not read from remote repository. Please make sure you have the correct access rights and the repository exists.

**Comment by TimaRebesh** [comment]
This issue is almost a year old. Are you going to solve it in the near future? Please, we are still waiting for

**Comment by iampstew** [comment]
Has there been any future update to this? Cause this is exactly what I've been looking for support for accessibility and me and other devs use the WAVE tool amongst other tools and this is what we are seeing.
